
Currently Loaded Modules:
  1) cmake/3.10.2                4) java/1.8.0_45     7) hdf5-parallel/1.8.16
  2) gcc/6.4.0                   5) cudatoolkit/9.1
  3) python/3.5-anaconda-4.1.1   6) openmpi/2.1.2

/usr/projects/p18_ebhlight3d/bolt_env/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Traceback (most recent call last):
  File "main.py", line 28, in <module>
    nls = nonlinear_solver(system)
  File "/users/brryan/github/Bolt/bolt/lib/nonlinear/nonlinear_solver.py", line 401, in __init__
    (af.flat(self.f)).to_ndarray(self._local_f_array)
AttributeError: 'Array' object has no attribute 'to_ndarray'
